.env

```
@@ -2,6 +2,10 @@
 VERSION="0.0.2"
 CEPH_VERSION="17.2.6"
 SPDK_VERSION="23.01"
+QUAY_SPDK="quay.io/ceph/spdk"
+QUAY_CEPH="quay.io/ceph/vstart-cluster"
+QUAY_NVMEOF="quay.io/ceph/nvmeof"
+QUAY_NVMEOFCLI="quay.io/ceph/nvmeof-cli"
```

Interesting refactoring. Given that it might be interesting to use an alternative container registry (e.g.: ghcr.io), I'd go 1 step forward:
```suggestion
CONTAINER_REGISTRY="quay.io"
QUAY_SPDK="${CONTAINER_REGISTRY}/ceph/spdk"
QUAY_CEPH="${CONTAINER_REGISTRY}/ceph/vstart-cluster"
QUAY_NVMEOF="${CONTAINER_REGISTRY}/ceph/nvmeof"
QUAY_NVMEOFCLI="${CONTAINER_REGISTRY}/ceph/nvmeof-cli"
```

---

Thanks @epuertat,
Updated! 
I included also the ceph level to CONTAINER_REGISTRY parameter.
If ceph part will not be included in the alternative container registry the code will be updated in one place only.

---

Ok, makes sense. In fact, many times developers push images to their personal accounts (e.g.: `quay.io/barakda`) instead of to the Ceph organization.